### PR TITLE
Added equilibrium and EoS fitting functions to tools.py

### PR DESCRIPTION
**Pull request by bobmyhill**
It's about time these workhorse functions were added!

### CODE REVIEW

File: burnman/tools.py [hunk 1]
```diff
@@ -93,3 +94,74 @@ def molar_volume_from_unit_cell_volume(unit_cell_v, z):
...
+        return volumes
+
+    popt, pcov = curve_fit(fit_data, PT, V,  guesses, V_sigma)
+    
+    return popt, pcov
+
+
+def equilibrium_pressure(minerals, multiplicities, temperature, guess=1.e5):
+    """
+    Takes a list of minerals, reaction multiplicities, a pressure and
```
Comment (tjhei) [inline burnman/tools.py]: where is the pressure argument?

Comment (tjhei) [inline burnman/tools.py]: can you rename the ```guess``` argument to something like ```temperature_initial_guess```?

**Comment by tjhei** [comment]
Can you please:
1. add a small test for each of the functions
2. document the parameters (types, units)

Thanks.

**Comment by bobmyhill** [comment]
Ok, tests and documentation added. This should be ready to roll!

**Comment by sannecottaar** [comment]
Me likes!

**Comment by tjhei** [comment]
nice, thank you!

**Comment by tjhei** [comment]
@ian-r-rose, @lordkman any comments?

**Comment by lordkman** [comment]
This looks great to me! Is it useful to add in equilibrium coefficient determinations too?

**Comment by bobmyhill** [comment]
Eventually, I would like to have a "thermocalc" function, which does a full equilibrium assemblage calculation. I envisage lnK calculations becoming useful at the same time.

So yes, but not for this PR :)

**Comment by lordkman** [comment]
Let me know when you're going to start those calculations, I can probably help with those. Otherwise, merge this up!

**Comment by ian-r-rose** [comment]
This looks great to me. I especially like the error estimates!

**Comment by bobmyhill** [comment]
One more function that I think might be useful...

**Pull request merged**